grype/version/format.go

```diff
@@ -85,6 +85,8 @@ func FormatFromPkg(p pkg.Package) Format {
 	switch p.Type {
 	case syftPkg.ApkPkg:
 		return ApkFormat
+	case syftPkg.BitnamiPkg:
```

Comment:
This surprised me. I'll write a longer comment later, but I thought Bitnami had custom versions, like `7.4.2-12`, and would need a custom version object. Many ecosystems have a custom version object.